src/pkg/reg/adapter/harbor/base/adapter.go

```diff
@@ -205,6 +210,17 @@ func (a *Adapter) PrepareForPush(resources []*model.Resource) error {
 		}
 		log.Debugf("project %s created", project.Name)
 	}
+
+	// do filter for proxy cache projects.
+	for _, res := range resources {
+		paths := strings.Split(res.Metadata.Repository.Name, "/")
```

Comment:
use util.ParseRepository()

Reply:
`util.ParseRepository()` will return `a/b` when repository is `a/b/c`, the actual project name is `a`.

Reply:
It seems that the util.ParseRepository() is an unused function and can be removed